src/components/ui/ColorButtons.scss

```
@@ -0,0 +1,33 @@
+.colorbuttons{
```

```suggestion
.colorbuttons {
```
I would keep the formatting and put a space here.
(and separate the words with dashes)

---

refactored this!